### PR TITLE
WindowsServer 1903 - Building 1903 Container images

### PR DESCRIPTION
**Pull request by nileshkumar01**
Building Windows Sever 1903 to enable Azure DevOps to build
1903 container images via the Azure Devops pipeline as currently there is only  windows 1803 option.

### CODE REVIEW

**Comment by kaylangan** [comment]
:wave: Hi @nileshkumar01! Thanks for the contribution!

We plan to add a 1903 container image, but it will likely coincide with the deprecation of the 1803 container. In the future, we hope to have a semi-annual Windows container image to match the semi-annual releases.

Whenever we add a new image, we have to evaluate the capacity we have across all our geos and all the images. At the moment, we aren't ready to add 1903. We'll keep this PR open and will re-visit it when we're ready capacity-wise.

Thanks for the contribution and your patience!

**Comment by kaylangan** [comment]
Related to #802

**Comment by nileshkumar01** [comment]
@kaylangan Would you know the timelines for the deprecation of the 1803 container ?

**Comment by kaylangan** [comment]
Not yet. It wouldn't happen until the end of the year at the very earliest, though likely later.

**Comment by nileshkumar01** [comment]
> Not yet. It wouldn't happen until the end of the year at the very earliest, though likely later.

Thank you for the info

File: images/win/scripts/Installers/WindowsContainer1903/Initialize-VM.ps1 [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+Write-Host $expandResult
+
+Write-Host "Disk sizes after expansion"
+
+wmic logicaldisk get size,freespace,caption
+
+# Adding description of the software to Markdown
+
+$Content = @"
+# Azure Pipelines Windows Container 1803 image
```
Comment (mozts2005) [inline images/win/scripts/Installers/WindowsContainer1903/Initialize-VM.ps1]:
```suggestion
# Azure Pipelines Windows Container 1903 image
```

File: images/win/scripts/Installers/WindowsContainer1903/Initialize-VM.ps1 [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+The following software is installed on machines in the Azure Pipelines **Windows Container 1803** pool.
```
Comment (mozts2005) [inline images/win/scripts/Installers/WindowsContainer1903/Initialize-VM.ps1]:
```suggestion
The following software is installed on machines in the Azure Pipelines **Windows Container 1903** pool.
```

File: images/win/WindowsContainer1903-Readme.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
+# Azure Pipelines Windows Container 1803 image
```
Comment (mozts2005) [inline images/win/WindowsContainer1903-Readme.md]:
```suggestion
# Azure Pipelines Windows Container 1903 image
```

**Pull request closed without merging**